web/src/beta/components/RadioGroup/RadioBox/index.tsx

```diff
@@ -7,23 +7,31 @@ export type Props = {
   selected?: boolean;
   value: string;
   onClick?: (value: string) => void;
+  label?: string;
 };
 
-const RadioBox: React.FC<Props> = ({ inactive, selected, value, onClick }: Props) => {
+const RadioBox: React.FC<Props> = ({ inactive, selected, value, label, onClick }: Props) => {
```

Comment:
the `inactive` prop doesn't seem to be used in RadioGroup. Please remove the logic around it

Reply:
done

Reply:
Please also remove it from Props type 🙏🏻

Reply:
done